src/packagedcode/models.py

```diff
@@ -529,6 +529,55 @@ def update(self, other):
         return self
 
 
+wheel = "wheel"
+sdist = "sdist"
+
+RELEASE_TYPES = (
+    None,
+    wheel,
```

Comment:
It feels strange to have package-specific release types in the base models.py file.